charts/astronomer/templates/commander/commander-role.yaml

```diff
@@ -24,15 +25,20 @@ metadata:
   namespace: {{ $namespaceName }}
   {{- end }}
   name: {{ $.Release.Name }}-commander
+  {{- if not $.Values.global.disableManageClusterScopedResources }}
```

Comment:
can we know why this condition is added here

Reply:
there are created by the installer service account which is provided at the time of installation of platform